### DISCUSSION TITLE
Change the position of skip and back button

### DISCUSSION THREAD

**Discussion by gaurav0909-max** [gilbarbara/react-joyride#1081] [Q&A] [answered]
Hey devs,
I want to change the position of skip and back button.
I want order like first back and then skip and next side by side.
Anyone does this in react-joyride? or have any idea ?
I would be great help to me 
Thanks in advance
@gilbarbara please check out and response to grow together

**Comment by gilbarbara** [comment, marked as answer]
Hey @gaurav0909-max 

You'll need to use a custom tooltip component to do this.
Check the [docs](https://docs.react-joyride.com/custom-components).
